keyboards/darkproject/kd83a_bfg_edition/config.h

```diff
@@ -0,0 +1,146 @@
+/* Copyright 2022 GSKY <gskyGit@gsky.com.tw>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+#define RGB_DISABLE_WHEN_USB_SUSPENDED
+#define EXTERNAL_FLASH_SIZE (8 * 1024)
+#define EEPROM_SIZE EXTERNAL_FLASH_SIZE
+
+/* USB Device descriptor parameter */
+#define DEVICE_VER 0x0001
+#define VENDOR_ID 0x2442
```

Comment:
This vendor ID is in use, and should be changed. 

https://www.the-sz.com/products/usbid/index.php?v=0x2442&p=&n=

In generally, anything part 0x4000 is available, but the higher it is, the better.

Reply:
changed vendor ID